snippets/country-localization.liquid

```diff
@@ -32,10 +34,10 @@
             data-value="{{ country.iso_code }}"
           >
             <span class="localization-form__currency">
+              {{ country | image_url: width: 18 | image_tag }}
               {{- country.currency.iso_code }}
-              {{ country.currency.symbol }} |</span
-            >
-            {{ country.name }}
+            </span>
+            | {{ country.name }}
```

Comment:
I agree that it would look better like in the screenshot for Andorra 👍 

![](https://screenshot.click/09-26-yztef-g7jm9.png)

Reply:
I agree with Ludo that the country flag looks visually better!